py/desisim/scripts/quickquasars.py

```diff
@@ -574,14 +603,14 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
             ('BRICKNAME', (str,8))]
         zbest = Table(np.zeros(nqso, dtype=columns))
         zbest["CHI2"][:]      = 0.
-        zbest["Z"]            = metadata['Z']+dz_fog
+        zbest["Z"]            = metadata['Z']
         zbest["ZERR"][:]      = 0.
 
         if args.sigma_kms_zfit:
            log.info("Added zfit error with sigma {} to zbest".format(args.sigma_kms_zfit))
-           sigma_zfit=(args.sigma_kms_zfit/299792.458)*(1.+metadata['Z'])
-           zbest["Z"]+=sigma_zfit*np.random.normal(0,1,nqso)
-           zbest["ZERR"]=sigma_zfit
+           sigma_zfit=mod_cauchy(loc=0,scale=args.sigma_kms_zfit,size=nqso,cut=3000)
+           zbest["Z"]+=sigma_zfit/c*(1.+metadata['Z'])
+           zbest["ZERR"]=args.sigma_kms_zfit/c*(1.+metadata['Z'])
```

Comment:
So sigma_kms_zfit used to be the Gaussian RMS of the redshift error. Now that it is a variable of the Lorentzian, is it still the case that the redshift error is described by this? It could be, I am just curious.

Reply:
Hi @andreufont, it is a good question. Formally it is not, since for the Cauchy distribution the variance is not well defined.  I thought that args.sigma_kms_zfit is still defining the width of the distribution, and in that sense the error (difference) to the true redshift. But is true it is underestimated. Be could compute the RMS directly on the generated samples, and read it from them, if you think this will be important...

Reply:
I don't know what is the best thing to do, but I wouldn't worry too much, since not many codes will actually use zbest["ZERROR"] unless they are actually using results from Redrock (in which case we would not run with this option).

So probably, the best thing to do is to either not write that column, or write ZERR=0 or ZERR=NAN, just to make it clear that none should really believe this number. 

It was cleaner when using the Gaussian distribution, but if it is not easy to get a number now, I would just forget about it.